DESCRIPTION

```diff
@@ -52,7 +52,8 @@ Imports:
     utils,
     parallel,
     grDevices,
-    backports
+    backports,
+    digest
```

Comment:
which aspects make digest imports? could it also be suggests with the usual brms check if the package is installed when needed?